src/components/avatar-audio-source.js

```diff
@@ -121,6 +128,17 @@ AFRAME.registerComponent("avatar-audio-source", {
     };
     APP.store.addEventListener("statechanged", this.onPreferenceChanged);
     this.el.addEventListener("audio_type_changed", this.createAudio);
+    APP.hubChannel.addEventListener("permissions_updated", this.onPermissionsUpdated);
+  },
+
+  onPermissionsUpdated() {
```

Comment:
If a user loses owner role they should not be in `moderatorAudioSource` anymore.

Reply:
Ops, I missed that. I've added the condition branch.